rfc/src/rfcs/0007-global-conditions.md

```diff
@@ -0,0 +1,111 @@
+- **Feature Name:** Global Conditions (`global-conditions`)
+- **Feature Request Issue:** <https://github.com/model-checking/kani/issues/2525>
+- **RFC PR:** <https://github.com/model-checking/kani/pull/2516>
+- **Status:** *Unstable*
+- **Version:** 0
+- **Proof-of-concept:** <https://github.com/model-checking/kani/pull/2532>
+
+-------------------
+
+## Summary
+
+A new section in Kani's output to summarize the status of properties that depend on other properties. We use the term *global conditions* to refer to such properties. 
+
+## User Impact
+
+The addition of new options that affect the overall verification result depending on certain property attributes demands some consideration.
+In particular, the addition of a new option to fail verification if there are uncoverable (i.e., unsatisfiable or unreachable) `cover` properties (requested in [#2299](https://github.com/model-checking/kani/issues/2299)) is posing new challenges to our current architecture and UI.
+
+This concept isn't made explicit in Kani, but exists in some ways.
+For example, the `kani::should_panic` attribute is a global condition because it can be described in terms of other properties (checks).
+The request in [#2299](https://github.com/model-checking/kani/issues/2299) is essentially another global conditions, and we may expect more to be requested in the future.
+
+In this RFC, we propose a new section in Kani's output focused on reporting global conditions.
+The goal is for users to receive useful information about hyperproperties without it becoming overwhelming.
```

Comment:
is "hyperproperties" meant as in "a property that relates two or more executions of the same program?"